modules/data.land/inst/doanload_NEON_multi_site.R

```diff
@@ -0,0 +1,111 @@
+#' Title generalized for download NEON files of multiSites. and automatically detect the most recent available NEON data and if data is problematic, try next year.
+#'
+#' @param settings a multisite settings object, should work with a single site setting. 
+#' @param main_store_dir the preferred path to store the downloaded NEON files. the log file and VEG_INFO should also be stored in that path
+#'
+#' @return VEG_INFO: a list object containing veg_info, site IDs of each site.
+#' @export log.txt file describing the progress of download
+
+Download_multiSites_NEON <- function(settings, main_store_dir){
```

Comment:
generalize name to make it clear that you are downloading VEG data, not that this is a generic NEON download function

Reply:
fixed

Reply:
function name still gives now indication that it downloads veg data. please fix.